addons/mass_mailing/static/src/js/mass_mailing_html_field.js

```diff
@@ -317,6 +317,8 @@ export class MassMailingHtmlField extends HtmlField {
         const editableAreaIsEmpty = value === "" || value === blankEditable;
 
         if (editableAreaIsEmpty) {
+            // unfold even on mobile to hide the toolbar
+            this.wysiwyg.snippetsMenu.setFolded(false);
```

Comment:
Yes, I need the toolbar to not show up until the user has selected a mailing theme. Otherwise they get a toolbar over a selection menu, which makes no sense.

Another option would be to set the toolbar to d-none but I feel like hiding/unhiding DOM elements manually all over the place makes things less easy to track in the code. But I get that it might not make much conceptual sense.